### PR TITLE
4 Responsive design

### PR DESCRIPTION
**Pull request by Lenninlasd**
## What does this PR do?

Implement responsive design.
### Screenshot

screen shot 2016-08-31 at 15 00 38
### Related Issue

[4](https://github.com/terranodo/angular-search/issues/4)

### CODE REVIEW

**Comment by milafrerichs** [comment]
Hi Lennin,

Jeff asked me to review your PR. 
I'll start today and finish it by tomorrow. 
Is there no travis set up yet?

File: app/components/datepicker/datepicker.html [hunk 1]
```diff
@@ -1,55 +1,52 @@
...
-    <div class="col-md-1">
-        <span class="input-group-btn searchbtn">
-          <button class="btn btn-default btn-sm infobtn" type="button" ng-click="showInfo()">
-            <span class="glyphicon glyphicon-info-sign"></span>
-        </button>
-        </span>
+    <div class="col-md-12 component-padding">
+        <div class="row">
+          <div class="col-md-12">
+              <form ng-submit="onSubmitDateText()" style="display: inline;">
```
Comment (milafrerichs) [inline app/components/datepicker/datepicker.html]: try not to use inline styles. move them into css

File: app/components/datepicker/datepicker.html [hunk 2]
```diff
@@ -1,55 +1,52 @@
...
+                  <div class="form-inline" style="display: inline;">
```
Comment (milafrerichs) [inline app/components/datepicker/datepicker.html]: try not to use inline styles. move them into css

File: app/view/exportButton.html [hunk 3]
```diff
@@ -1,28 +1,24 @@
-<div ng-controller="ExportController">
-    <div class="col-md-2">
-        <input type="number" class="ng-valid ng-not-empty ng-dirty ng-valid-number ng-touched" ng-model="export.numDocuments">
+<div class="row" ng-controller="ExportController">
+    <div class="col-sm-4 col-xs-3" style="padding-top:30px">
```
Comment (milafrerichs) [inline app/view/exportButton.html]: try not to use inline styles. move them into css

File: app/view/exportButton.html [hunk 4]
```diff
@@ -1,28 +1,24 @@
...
+        <input type="number" class="form-control input-sm ng-valid ng-not-empty ng-dirty ng-valid-number ng-touched" ng-model="export.numDocuments">
     </div>
-    <div class="col-md-9">
+
+    <div class="col-sm-8 col-xs-9" style="padding-top:30px">
```
Comment (milafrerichs) [inline app/view/exportButton.html]: try not to use inline styles. move them into css

File: app/view/exportButton.html [hunk 5]
```diff
@@ -1,28 +1,24 @@
...
     <div class="col-md-12">
-        <div class="form-horizontal">
-            <div class="form-group">
-                <label for="focusedInput" class="control-label col-md-2 col-md-offset-9">
+            <div class="pull-right">
+                <label for="focusedInput" class="control-label">
                     Results: <span ng-bind="export.numDocuments"></span> of <span ng-bind="export.options.ceil"></span>
                 </label>
-                <div class="col-md-1">
+                <span class="">
```
Comment (milafrerichs) [inline app/view/exportButton.html]: remove the empty class attribute

File: app/view/toolbarSearchField.html [hunk 6]
```diff
@@ -1,32 +1,26 @@
-<div ng-controller="SearchController">
-    <div class="col-md-9">
+<div ng-controller="SearchController" style="width:400px;">
```
Comment (milafrerichs) [inline app/view/toolbarSearchField.html]: try not to use inline styles. move them into css

File: app/view/toolbarSearchField.html [hunk 7]
```diff
@@ -1,32 +1,26 @@
...
+    <div class="col-xs-7" style="padding-left: 4px;padding-right: 4px;">
```
Comment (milafrerichs) [inline app/view/toolbarSearchField.html]: try not to use inline styles. move them into css

File: app/view/toolbarSearchField.html [hunk 8]
```diff
@@ -1,32 +1,26 @@
...
+            <input class="form-control input-sm" type="text" placeholder="Enter keyword" ng-keypress="onKeyPress($event)"
+              ng-model="searchInput" ng-click="onFocus()" ng-blur="onBlur()">
+            <span class="input-group-addon">
+                <i class="glyphicon glyphicon-search"></i>
+            </span>
          </div>
     </div>
- <div class="col-md-1">
-         <span class="searchbtn">
+    <div class="col-xs-5" style="padding-left: 4px;padding-right: 4px;">
```
Comment (milafrerichs) [inline app/view/toolbarSearchField.html]: try not to use inline styles. move them into css

File: app/view/userFilter.html [hunk 9]
```diff
@@ -1,20 +1,20 @@
...
-                <span class="input-group-addon">
-                    <i class="glyphicon glyphicon-search"></i>
-                </span>
-            </div>
-        </form>
+    <div class="row">
+        <div class="col-xs-11">
+            <form ng-submit="userSearch()">
+                <div class="input-group searchinputfield" id="userfilterinput">
+                    <input class="form-control input-sm" type="text" placeholder="@user" style="display:inline"
```
Comment (milafrerichs) [inline app/view/userFilter.html]: try not to use inline styles. move them into css

File: index-dev.html [hunk 10]
```diff
@@ -55,52 +55,68 @@
 </head>
 
 <body ng-controller="MainController">
-    <div class="container-fluid">
+    <div class="container-fluid" style="background: #eee;">
```
Comment (milafrerichs) [inline index-dev.html]: try not to use inline styles. move them into css

File: index.html [hunk 11]
```diff
@@ -31,37 +31,66 @@
 </head>
 
 <body ng-controller="MainController">
-    <div class="container-fluid">
+    <div class="container-fluid" style="background: #eee;">
```
Comment (milafrerichs) [inline index.html]: try not to use inline styles. move them into css

**Comment by Lenninlasd** [comment]
Hi @milafrerichs, thanks for the feedback, travis is not setup yet.

**Comment by milafrerichs** [comment]
@jj0hns0n or @ingenieroariel can we setup travis for this one?
the `.travis.yml` is already there. I don't have the rights to do it for this one.

maybe I can get access to the organisation/repo as well. Would like to setup [hound](https://houndci.com) which would help with reviewing the PR's and finding styling/linting mistakes.

**Pull request merged**